js/libdeno.ts

```diff
@@ -20,6 +28,21 @@ interface Libdeno {
     ) => void
   ) => void;
 
+  setPromiseRejectHandler: (
+    handler: (
+      error: Error,
+      event: number,
```

Comment:
How about about 
```ts
event: "RejectWithNoHandler" | "HandlerAddedAfterReject" | "ResolveAfterResolved" | "RejectAfterResolved"
```

Reply:
Event types from v8 here are integers. Not quite sure if we want to covert to string reprs

Reply:
The only concern would be about performance.. presumably numbers are faster.
Is this hot code?

Reply:
The handler is invoked twice for each caught promise reject. Not quite sure if it is hot enough

Reply:
I think mapping the V8 numbers to strings in C++ isn't so bad. This will simply the libdeno JS interface quite a bit.

Reply:
```c++
const char* PromiseRejectStr(enum v8::PromiseRejectEvent e) {
  switch (e) {
    case kPromiseRejectWithNoHandler:
      return "RejectWithNoHandler";
    case kPromiseHandlerAddedAfterReject:
      return "HandlerAddedAfterReject";
    case kPromiseResolveAfterResolved:
      return "ResolveAfterResolved";
    case kPromiseRejectAfterResolved:
      return "RejectAfterResolved";
  }
```

Reply:
Thanks! Added the function.